src/lbaf/Model/lbsObject.py

```diff
@@ -163,3 +166,11 @@ def set_communicator(self, c) -> None:
     def get_subphases(self) -> list:
         """Return subphases of this object."""
         return self.__subphases
+
+    def set_unused_params(self, unused_params: dict):
+        """Assign any extraneous parameters."""
+        self.__unused_params = unused_params
+
+    def get_unused_params(self) -> dict:
```

Comment:
Due to the way we dynamically get all Object QOI, this will now list `unused_params` as a QOI